### ISSUE TITLE
Can't install it!

### ISSUE THREAD

**Issue by nitroos-ai** [sebaro/ViewTube#3]
Hello, I can't install the script in firefox 97.0.2 with greasemonkey 4.11.

When I click on the script to install it, a new window shows up with this message:

parseUserScript() got no content!

![ViewTube](https://user-images.githubusercontent.com/68450737/157721924-20c2cf46-0051-4626-99de-444360448108.png)

**Comment by sebaro** [comment]
Try the direct link:
https://github.com/sebaro/ViewTube/raw/master/viewtube.user.js
https://github.com/sebaro/ViewTube/raw/master/viewtubetest.user.js

Or create a new script with GM and paste the content.

**Comment by nitroos-ai** [comment]
Thanks, both links works fine. I installed the first link.
May be you should update the link on your website with the one from github.

**Comment by tnbp** [comment]
Unfortunately, that's exactly my experience too: github works, sebaro.pro doesn't. Given the choice, I would prefer sebaro.pro.

**Comment by sebaro** [comment]
> Unfortunately, that's exactly my experience too: github works, sebaro.pro doesn't. Given the choice, I would prefer sebaro.pro.

What is the problem with the installation?

**Comment by tnbp** [comment]
> > Unfortunately, that's exactly my experience too: github works, sebaro.pro doesn't. Given the choice, I would prefer sebaro.pro.
> 
> What is the problem with the installation?

Same as https://github.com/sebaro/ViewTube/issues/3#issue-1165522637

**Comment by sebaro** [comment]
I need more info, which browser and what version.

For me it's working with FF 133.0.3 (64-bit) and GM 4.13.

![screenshot-2024-12-22 20:56](https://github.com/user-attachments/assets/d21c92a1-f363-4928-9092-2ef52f78fa57)

**Comment by tnbp** [comment]
Same version of Firefox *and* Greasemonkey. 
![grafik](https://github.com/user-attachments/assets/2d9351bb-bd36-409a-987d-f5e899e454cd)
This may actually not be a problem with ViewTube itself, but Greasemonkey.

**Comment by sebaro** [comment]
The error means the browser can't read the script's content:

`window.parseUserScript = function(content, url, failWhenMissing=false) {
  if (!content) {
    throw new Error('parseUserScript() got no content!');
  }`

Do you use scripts/addons or settings that force https?

**Comment by tnbp** [comment]
> Do you use scripts/addons or settings that force https?

Yes, good catch. That's probably the issue: I force Firefox to use HTTPS:
![grafik](https://github.com/user-attachments/assets/1ee05183-8cf7-4ab9-9f69-97ce4038ab54)
Disabling that fixes the issue.
